frontend/src/App.js

```diff
@@ -53,16 +53,16 @@ function App() {
         <ThemeToggle />
       </Header>
 
-      <Content className="content">
```

Comment:
@Snuzzn  I removed content because it was affecting the style of other pages. I also removed  `.content` in your progression and term planner `.less` so the styling is still applied.